pkg/stub/handler.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/sirupsen/logrus"
+
```

Comment:
is there a reason not to use "github.com/golang/glog"?

Reply:
I was only using logrus because the SDK uses it; no objection to switching to glog for k8s/origin consistency (although for the record I don't like glog at all). I can switch in a followup if everyone agrees on glog.